src/shogun/base/SGObject.h

```diff
@@ -298,30 +310,20 @@ class SGObject: public std::enable_shared_from_this<SGObject>
 	 */
 	template <typename T,
 		      typename std::enable_if_t<!is_string<T>::value>* = nullptr>
-	void put(const Tag<T>& _tag, const T& value) noexcept(false)
```

Comment:
removed these as they don't do anything for us. This is just useful if the template type determines if it is noexcept